src/server/appHandler.js

```diff
@@ -82,8 +82,10 @@ function renderApp (res, renderProps, history, store) {
     )
 
     let state = store.getState()
+    let unreadCount = get(state, 'people.current.new_notification_count')
 
     return React.createElement(Html, {
+      pageTitle: 'Hylo' + (unreadCount && ` (${unreadCount})`),
```

Comment:
this is not updating when the store updates, because it's outside Redux. Any thoughts on how to get this updating? One possibility would be doing it with JS in eg the App component